components/navigation/navigation.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>nav</title>
+    <link rel="stylesheet" href="../css/main.css" />
+    <link rel="stylesheet" href="navigation.css">
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css"
+      integrity="sha512-iBBXm8fW90+nuLcSKlbmrPcLa0OT92xO1BIsZ+ywDWZCvqsWgccV3gFoRBv0z+8dLJgyAHIhR35VZc2oM/gI1w=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+  </head>
+  <body>
+
+    <header class="top-nav nav-bg-primary ">
+        <div class="">
+          <button class="btn-md"><i class="fas fa-bars  nav-text-primary"></i></button>
+          <span class="h-md nav-text-primary">Title</span>
```

Comment:
The naming is not clear for `h-md`

Reply:
I have made corrections as per your suggestion
thanks for reviewing